src/app/modules/users/login/login.component.css

```diff
@@ -1,4 +1,31 @@
+::ng-deep mat-form-field {
+  color: white !important;
```

Comment:
Don't use !important in CSS. Consult with @katarina-glorija if you are having difficulties with CSS.